client/src/pages/ForumPage/ForumPageDiscover.js

```diff
@@ -9,18 +9,38 @@ import Button from "../../components/Button/Button";
 import Footer from "../../components/Footer/Footer";
 import Pill from "../../components/Pill/Pill";
 import InputSelect from "../../components/InputSelect/InputSelect";
+import getHashtags from "../../services/Forum.js";
+import { Link } from "react-router-dom";
+
 
 const handleSelect = props => {
-  console.log(props);
+  
 };
 function pillClick(val) {
-  console.log(val);
+  
 }
 const handleClick = e => {
-  console.log("Clicked");
+ 
 };
 
-const ForumPageDiscover = props => {
+
+class ForumPageDiscover extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      hashtags: [],
+    }
+    this.getHashtags = getHashtags.bind(this);
+  }
+
+  componentDidMount() {
+  this.getHashtags()
+}
+
+render() {
+
+  const hashtags = this.state.hashtags ? this.state.hashtags: [];
```

Comment:
The formatting here is really weird - can you make sure you have Prettier installed and set up to auto format on paste and on save?

Reply:
Double-checked and I do... manually changed the formatting